Tests/WinAppDriverTests/RequestsTests.swift

```diff
@@ -96,6 +101,39 @@ class RequestsTests: XCTestCase {
         try XCTAssert(!Self.hasKeyboardFocus(app.findWhatEditBox))
     }
 
+    func testSessionSendKeys_scopedModifiers() throws {
+        try app.findWhatEditBox.click()
+        try app.session.sendKeys(Keys.shift(Keys.a) + Keys.a)
+        XCTAssertEqual(try app.findWhatEditBox.text, "Aa")
```

Comment:
How does the text get reset between tests?

Reply:
I'm recreating the app every time now to isolate tests